readme.md

```diff
@@ -17,6 +17,20 @@ Using jQuery, write code that makes an AJAX get request to "http://kittengifs.co
 
 Your Answer:
 ```js
+function getKittens(){
+  var url = "http://kittengifs.com/gifs";
+  $.ajax({
+      url: url,
+      type: "get",
+      dataType: "json"
+    }).done(function(response){
+      console.log(response.gifs.count);
+    }).fail(function(){
+      console.log("Ajax request failed!");
+    }).always(function(){
+      console.log("Something happened but I'm not going to tell you what.");
+    });
+}
```

Comment:
:+1:  
Since the server "responds with a JSON array of objects", `response.length` would suffice.